daemon/inertiad/token.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/ubclaunchpad/inertia/daemon/inertiad/crypto"
+)
+
+// tokenHandler generates a new token
+func tokenHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
same here, this one actually isn't that useless to test. refer to other handler for how you might go about setting this one up for tests